src/core/mantle/types/appearance.ts

```diff
@@ -139,6 +140,16 @@ export type ResourceAppearance = {
   clampToGround?: boolean;
 };
 
+export type RasterAppearance = {
+  url?: string;
+  type?: "wms";
```

Comment:
```suggestion
```

Type should be handled on `data`. Type is not a "style" and "appearance" of raster.

Reply:
I fixed it, also the comment below is fixed.
[fix: data type](https://github.com/reearth/reearth-web/pull/387/commits/3444ed9819ace9e93a84837595fd4d3ed59f52e3)